service/builder/exporters_builder.go

```diff
@@ -32,6 +32,26 @@ type builtExporter struct {
 	me exporter.MetricsExporter
 }
 
+// Start the exporter.
+func (exp *builtExporter) Start(host exporter.Host) error {
+	var errors []error
+	if exp.te != nil {
+		err := exp.te.Start(host)
+		if err != nil {
+			errors = append(errors, err)
+		}
+	}
+
+	if exp.me != nil {
+		err := exp.te.Start(host)
```

Comment:
```suggestion
		err := exp.me.Start(host)
```

Reply:
@songy23 thanks for catching the bug! I added a test to verify this.